package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "watch": "node bin/watch",
-    "test": "standard && hallmark && node test/unit",
```

Comment:
I got a weird error on Jenkins https://ci.delta.chat/blue/organizations/jenkins/deltachat-desktop-linux/detail/PR-552/4/pipeline .. disabling for now to avoid rabbit hole / side quest .. doesn't happen locally or not on Travis